Common/src/main/java/dk/sdu/group/one/data/Vector2.java

```diff
@@ -21,6 +21,9 @@ public Vector2(float x, float y) {
      * @return the normalized vector
      */
     public Vector2 normalize() {
+        // If vector is a zero vector
+        if (this.x == 0 && this.y == 0) return this;
+
         double l = Math.pow(x, 2) + Math.pow(y, 2);
```

Comment:
please be more descriptive. Spend a solid 30 seconds wonderign why you're sqrt(1)'ing.....

Reply:
Resolved as of 3d2aaed27e4535e1bedeb1f05c6aee365a743477